packages/wallets/src/features/cfd/modals/CTraderSuccessModal/CTraderSuccessModal.tsx

```diff
@@ -82,13 +82,17 @@ const CTraderSuccessModal = ({ createdAccount, isDemo, walletCurrencyType }: TCT
                 marketType='all'
                 platform={PlatformDetails.ctrader.platform}
                 title={
-                    <Localize
-                        i18n_default_text='Your {{ctraderTitle}}{{demoTitle}} account is ready'
-                        values={{
-                            ctraderTitle: PlatformDetails.ctrader.title,
-                            demoTitle: isDemo ? localize('demo') : '',
-                        }}
-                    />
+                    isDemo ? (
+                        <Localize
```

Comment:
@lubega-deriv could we somehow remove duplicated content for mobile/desktop?
maybe create a title variable

Reply:
Updated